crates/runtime/src/config.rs

```diff
@@ -11,6 +12,12 @@ pub struct RuntimeConfig {
     /// Path where conductor data is stored
     pub data_root_path: PathBuf,
 
+    /// Network config
+    pub network: RuntimeNetworkConfig,
```

Comment:
The network config is split out in a separate type, so that it can be specified by the runtime distributor.